configs/remediations.json

```diff
@@ -0,0 +1,22 @@
+{
+  "roles": [
+    {
+      "name": "Remediations Access",
+      "description": "A role which grants read and write permissions.",
+      "system": true,
+      "platform_default": true,
+      "version": 4,
```

Comment:
It doesn't _really_ matter, but this doesn't need to be 4, it could be 2. The seeding versions technically start at 2 and bump up from there. Again, it doesn't make a huge difference, as you'd just be incrementing from 4 instead of 2 if you need to make changes to this role in the future. We don't store the version, just check for a diff.

Reply:
Oh I see. I saw other roles using 4 so I did the same but I'll change this to 2.